Packs/Gmail/Integrations/Gmail/Gmail.py

```diff
@@ -1805,6 +2034,8 @@ def fetch_incidents():
             incidents.append(incident)
 
     demisto.info('extract {} incidents'.format(len(incidents)))
+    if params.get('isFetch'):
+        return None
```

Comment:
```suggestion
        return []
```

Reply:
we have taken reference from another PR review: [https://github.com/demisto/content/pull/8806#discussion_r487512384](https://github.com/demisto/content/pull/8806#discussion_r487512384)

And this integration.
https://github.com/demisto/content/blob/master/Packs/PaloAltoNetworks_IoT/Integrations/PaloAltoNetworks_IoT/PaloAltoNetworks_IoT.py#L506